### PR TITLE
Update main.c

### PR DESCRIPTION
**Pull request by SpeechKingDusha**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -1,4 +1,4 @@
 int is_Lacomij_kusochek (int one_part, int second_part);
 unsigned long long Krasavchik;
-
+BigKrasavchik_t bigKrasavchik;
```
Comment (IgorMoiseev1988) [inline main.c:3]: Where defined this type? Or you wont use undeclared type?

Comment (Notka554) [reply main.c:3]: ku ku

Comment (IgorMoiseev1988) [reply main.c:3]: You are need to define this type before use.

**Review by Tigran-C-creat — CHANGES_REQUESTED** [review]
it is nice

File: main.c (hunk 1 above)
Comment (Notka554) [reply main.c:3]: I don't now this type

Comment (Notka554) [reply main.c:3]: know*

File: main.c [hunk 2]
```diff
@@ -1,4 +1,9 @@
-int is_Lacomij_kusochek (int one_part, int second_part);
-unsigned long long Krasavchik;
+#include <fucking.h>
```
Comment (IgorMoiseev1988) [inline main.c]: Unknown header

File: main.c [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
 
+int is_Lacomij_kusochek (int one_part, int second_part);
```
Comment (IgorMoiseev1988) [inline main.c:3]: Unused function, and it not defined

File: main.c [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
 
+int main ()
+{
+unsigned long long Krasavchik;
```
Comment (IgorMoiseev1988) [inline main.c:7]: Unused var

File: main.c [hunk 5]
```diff
@@ -1,4 +1,9 @@
...
+BigKrasavchik_t bigKrasavchik;
```
Comment (IgorMoiseev1988) [inline main.c:8]: 1) Unuesd var
2) Unknown type

File: main.c [hunk 6]
```diff
@@ -1,4 +1,9 @@
...
+}
```
Comment (IgorMoiseev1988) [inline main.c:9]: No return statement at end of non-void function

**Review by IgorMoiseev1988 — CHANGES_REQUESTED** [review]

File: main.c (hunk 5 above)
Comment (Notka554) [reply main.c:8]: ?

File: main.c [hunk 7]
```diff
@@ -1,4 +1,21 @@
+#include <stdio.h>
+#include "fucking.h"
```
Comment (IgorMoiseev1988) [inline main.c:2]: Unknown header

File: main.c [hunk 8]
```diff
@@ -1,4 +1,21 @@
...
 int is_Lacomij_kusochek (int one_part, int second_part);
+
+struct BigKrasavchik_t {
+	int size_left_hand;
+	int size_right_hand;
+};
+int main ()
+{
 unsigned long long Krasavchik;
+struct BigKrasavchik_t bigKrasavchik;
```
Comment (IgorMoiseev1988) [inline main.c:13]: Unused var

File: main.c [hunk 9]
```diff
@@ -1,4 +1,21 @@
...
+is_Lacomij_kusochek (135, 115);
```
Comment (IgorMoiseev1988) [inline main.c:14]: Ignore return value. May be you don't need call this functions?

File: main.c [hunk 10]
```diff
@@ -1,4 +1,21 @@
...
+}
```
Comment (IgorMoiseev1988) [inline main.c:15]: No return statement. Main declared as non void.

File: main.c [hunk 11]
```diff
@@ -1,4 +1,21 @@
...
 
 
+int is_Lacomij_kusochek (int one_part, int second_part)
+{
+ return one_part+second_part;
+}
```
Comment (IgorMoiseev1988) [inline main.c:21]: Back code-style : tabs must be same on all files

**Review by IgorMoiseev1988 — CHANGES_REQUESTED** [review]

**Review by IgorMoiseev1988 — CHANGES_REQUESTED** [review]
It is good

**Review by Notka554 — COMMENTED** [review]
Good

**Review by Tigran-C-creat — CHANGES_REQUESTED** [review]
it is good

**Review by Notka554 — APPROVED** [review]
Good

**Comment by Tigran-C-creat** [comment]
@SpeechKingDusha

**Comment by IgorMoiseev1988** [comment]
Pull request not complete

**Comment by SpeechKingDusha** [comment]
now

**Review by Tigran-C-creat — APPROVED** [review]

**Review by IgorMoiseev1988 — APPROVED** [review]

**Review by Tigran-C-creat — APPROVED** [review]

**Pull request merged**